packages/js/client/src/default-client-config.ts

```diff
@@ -29,8 +26,28 @@ export const getDefaultClientConfig = Tracer.traceFunc(
   "client-js: getDefaultClientConfig",
   (): ClientConfig<Uri> => {
     return {
-      envs: [],
-      redirects: [],
+      envs: [
+        {
+          uri: new Uri(defaultWrappers.graphNode),
+          env: {
+            provider: "https://api.thegraph.com",
+          },
+        },
+      ],
+      redirects: [
+        {
+          from: new Uri("wrap://ens/sha3.polywrap.eth"),
+          to: new Uri(defaultWrappers.sha3),
```

Comment:
@nerfZael would it be possible in the new URI resolution system to registry a redirect w/ a deprecation warning? This way when invocations are made to the old URI, the developer would know that the old URI is being phased out.

Reply:
Yea, it'd be simple to create a URI resolver that inherits from redirect resolver and console logs a message when resolving.
I'm not sure it'd be THAT useful since I think we should migrate to explicit Client configs instead of the current default/implicit one

Reply:
> I'm not sure it'd be THAT useful since I think we should migrate to explicit Client configs instead of the current default/implicit one

Yes! Agreed